tests/test_cmdline.py

```diff
@@ -122,43 +122,43 @@ def test_adduser_help(self, mycmd):
             *add*user*
         """)
 
-    def test_add_del_user(self, mycmd):
+    def test_add_del_user(self, mycmd, mailcow_domain):
         mycmd.run_ok(["add-token", "test1", "--expiry=1d", "--prefix", "pytest."])
-        addr = "pytest.%s@x.testrun.org" % (randint(0, 99999),)
+        addr = "pytest.%s@%s" % (randint(0, 99999), mailcow_domain)
         mycmd.run_ok(["add-user", addr], """
-            *Created*pytest*@x.testrun.org*
+            *Created*pytest*@*
         """)
         mycmd.run_ok(["list-users"], """
-            *pytest*@x.testrun.org*test1*
+            *pytest*@*test1*
         """)
```

Comment:
```suggestion
            *{addr}* 
        """.format(addr=addr))
```
With different addresses (bot, admin, supportuser) flying around in the tests it is good to be a bit more precise here.